test/e2e/tests/phishing-controller/mock-malicious-websocket-connection/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>WebSocket Test</title>
+    <link rel="icon" href="data:," />
+    <script>
+      class MockWebSocket {
+        constructor(url) {
+          this.url = url;
+          this.readyState = 0;
+
+          console.log(`MockWebSocket: Attempting to connect to ${url}`);
+
+          // Simulate connection behavior
+          setTimeout(() => {
+            try {
+              const { hostname } = new URL(url);
+              if (hostname === 'blocked.example.com') {
```

Comment:
It looks like this mock page is redirecting to the phishing page itself. It's not actually testing that MetaMask is redirecting.

You can tell it's not testing the C2 blocklist because if you remove the C2 blocklist from the test, it still passes

Reply:
We hopped on a call earlier this week and chatted about this. A believe he has a testing solution in progress that involves enabling websocket mocking in our fixtures